diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/NewlinesRule.kt

```diff
@@ -371,13 +372,15 @@ class NewlinesRule(private val configRules: List<RulesConfig>) : Rule("newlines"
         SECONDARY_CONSTRUCTOR -> parent(CLASS)!!
         else -> treeParent
     }
-        .getIdentifierName()?.text
+            .getIdentifierName()?.text
 
     private fun ASTNode.getOrderedCallExpressions(psi: PsiElement, result: MutableList<ASTNode>) {
         // if statements here have the only right order - don't change it
 
         if (psi.children.isNotEmpty() && (psi.children[0].node.elementType != DOT_QUALIFIED_EXPRESSION
-                        && psi.children[0].node.elementType != SAFE_ACCESS_EXPRESSION )) {
+                        && psi.children[0].node.elementType != SAFE_ACCESS_EXPRESSION)) {
+                            if (psi.children[0].node.elementType == POSTFIX_EXPRESSION)
```

Comment:
```suggestion
                            if (psi.firstChild.node.elementType == POSTFIX_EXPRESSION)
```
to not use explicit index access